frontend/javascripts/admin/team/team_list_view.tsx

```diff
@@ -112,6 +118,43 @@ class TeamListView extends React.PureComponent<Props, State> {
     );
   }
 
+  renderTeamRolesForUser(user: APIUser, highlightedTeam: APITeam) {
+    const tags = user.isAdmin
+      ? [["Admin - Access to all Teams", "red"]]
+      : user.teams
+          .filter((team) => team.id === highlightedTeam.id)
+          .map((team) => {
+            const roleName = team.isTeamManager ? "Team Manager" : "Member";
+            return [`${roleName}`, stringToColor(roleName)];
+          });
+
+    return tags.map(([text, color]) => (
+      <Tag key={`${text}_${user.id}`} color={color} style={{ marginBottom: 4 }}>
+        {text}
+      </Tag>
+    ));
```

Comment:
That was my first thought too. But there are slight differences. On this page, the `Dataset Manager` role is ignored/not display as it has no meaning for Teams. Also the the team name is also no part of the Tag, e.g. `Member` : `Default - Member`

Reply:
I see. How about moving the similar functions into the same module (could be either the team list view or user list view) so that they are next to each other? I think, it's better to keep them together, so that the chance of divergence is reduced.

Reply:
I move both render methods into one file. Please see commit https://github.com/scalableminds/webknossos/pull/6915/commits/c0546d355a2a2790633a70844cf7ae51ce27b430

Reply:
great :+1: feel free to merge.